### PR TITLE
GitHub Action - Cleanup on PR Close

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

**Comment by DerekRoberts** [comment]
Closes are triggering a cleanup! Success. :)

**Comment by DerekRoberts** [comment]
> Take a look at the build stage. It's actually picking up the wrong commits, sadly `git rev-parse --abbrev-ref HEAD` doesn't pick up the correct branch in GHA environment. And that's why I created a pr in `bcgov/openshift-launchpad-deployment`

@Kyubinhan 
Crap.  We're getting a ton of timeouts from cloning in OpenShift with `git describe --all --contains HEAD`.  Looks like something we'll have to dig into tomorrow.

**Comment by Kyubinhan** [comment]
> > Take a look at the build stage. It's actually picking up the wrong commits, sadly `git rev-parse --abbrev-ref HEAD` doesn't pick up the correct branch in GHA environment. And that's why I created a pr in `bcgov/openshift-launchpad-deployment`
> 
> @Kyubinhan
> Crap. We're getting a ton of timeouts from cloning in OpenShift with `git describe --all --contains HEAD`. Looks like master is being cloned, which we'll definitely have to figure out. How does your mod to Coco's entrypoint fix that?

It takes parameters as input. See this PR.
https://github.com/bcgov/openshift-launchpad-deployment/pull/5/files

File: .github/workflows/pr_close.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - name: Set Variables
+        shell: bash
+        run: |
+          echo "::set-env name=SUFFIX::pr-${{ github.event.number }}"
+      
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cleanup Script
+        uses: bcgov/openshift-launchpad-deployment@custom-script
```
Comment (Kyubinhan) [inline .github/workflows/pr_close.yml]: Can you use `@master` here as well

**Pull request merged**